### PR TITLE
Make airfoilRANSEdgeTrilinos restart test run on GPU.

### PR DESCRIPTION
**Pull request by overfelt**
**Pull-request type:**
- [ ] Bug fix 
- [ ] Documentation update
- [x] Feature enhancement

## Description of the pull-request

*Provide a description of your proposed changes. If there is a related issue, please specify.**

## Checklist

**NOTE** The checklist below is a suggestion and not mandatory. You don't have
to _check all boxes_ to submit a pull request. However, please add a brief
explanation in your pull request summary explaining the omission for the benefit
of reviewers and developers.

*All pull requests*
- [ ] Builds successfully (*must test on at least one system/compiler combination*)
  - Operating systems 
    - [x] Linux
    - [ ] MacOS
  - Compilers 
    - [x] GCC
    - [ ] LLVM/Clang
    - [ ] Intel compilers
    - [x] NVIDIA CUDA
- [ ] Compiles without warnings
- [ ] Passes all unit tests
- [ ] Passes all regression tests
- [ ] Documentation builds without errors

*For new code updates*
- [ ] Documentation updates - additions to user, theory, and verification manuals
- [ ] New unit tests providing coverage for new code, bug fixes etc.
- [ ] New regression tests exercising the new code

### CODE REVIEW

File: reg_tests/test_files/airfoilRANSEdgeTrilinos.rst/airfoilRANSEdgeTrilinos.rst.i [hunk 1]
```diff
@@ -68,17 +69,17 @@ realms:
 
         - WallDistance:
             name: myNDTW
-            max_iterations: 1
+            max_iterations: 2
```
Comment (sayerhs) [inline reg_tests/test_files/airfoilRANSEdgeTrilinos.rst/airfoilRANSEdgeTrilinos.rst.i:72]: @overfelt Why were these iterations increased?

File: reg_tests/test_files/airfoilRANSEdgeTrilinos.rst/airfoilRANSEdgeTrilinos.rst.i [hunk 2]
```diff
@@ -193,7 +193,7 @@ realms:
 
     output:
       output_data_base_name: results/du91w2.e
-      output_frequency: 5
+      output_frequency: 1
```
Comment (sayerhs) [inline reg_tests/test_files/airfoilRANSEdgeTrilinos.rst/airfoilRANSEdgeTrilinos.rst.i:196]: We should probably switch this back to a not output so many files during the regression test run

File: reg_tests/test_files/airfoilRANSEdgeTrilinos.rst/airfoilRANSEdgeTrilinos.rst.i (hunk 1 above)
Comment (overfelt) [reply reg_tests/test_files/airfoilRANSEdgeTrilinos.rst/airfoilRANSEdgeTrilinos.rst.i:72]: I was trying things to get the CPU/GPU results to match better and just increased all iterations on everything.  Not sure it it helped for this particular calculation.

File: reg_tests/test_files/airfoilRANSEdgeTrilinos.rst/airfoilRANSEdgeTrilinos.rst.i (hunk 2 above)
Comment (overfelt) [reply reg_tests/test_files/airfoilRANSEdgeTrilinos.rst/airfoilRANSEdgeTrilinos.rst.i:196]: OK.

**Pull request merged**